vipps-recurring-api.md

```diff
@@ -858,21 +864,21 @@ An example from a response:
 {
   "status": "FAILED",
   "type": "RECURRING",
-  "failureReason": "insufficient_funds",
-  "failureDescription": "Payment was declined by the payer bank due to lack of funds"
+  "failureReason": "user_action_required",
+  "failureDescription": "User action required"
 }
 ```
 
 Here is a list of possible values for `failureReason`, their respective descriptions and possible actions that the user/merchant could take.
 
 | Reason | Description | Action |
 | ---- | ----------- | ------ |
-| insufficient_funds | Payment was declined by the payer bank due to lack of funds. | User must either add funds to the card to cover the difference between the amount to be paid. Alternatively they can change to another, or add a new, payment source that is adequately funded to complete the transaction. |
-| invalid_card | The user tried to pay using a card that has either expired or is disabled by the issuer. | User must change, or add a new, payment source on the agreement in Vipps. |
-| verification_required | Payment declined because the issuing bank requires verification. | Ask the user to change, or add a new, payment source on their agreement in Vipps. Alternatively removing and then adding the card might solve the issue. |
-| invalid_payment_source | The provided payment source is disabled or does not exist. | User must change payment source for the agreement. |
+| user_action_required | Payment failed. Could be lack of funds, card is blocked for ecommerce, card is expired. | User will get notified in Vipps and need to take action. This could be to add funds to the card or change the card on the agreement. |
 | charge_amount_too_high | Amount is higher than the users specified max amount | The user have a lower `maxAmount` on the variableAmount agreement than the amount of the charge. The user must update their `maxAmount` on the agreement for the charge to be processed.
-| internal_error | Internal Error / Something went wrong | The error could not be identified as one of the above. Try to create the charge again, changing or adding payment sources on the agreement, or contact Vipps for more information. |
+| non_technical_error | Payment failed. Could be that the user has deleted their Vipps profile. | The user needs to take action in Vipps. |
+| technical_error | Payment failed due to a technical error in Recurring or a downstream service | As long as the charge is not in status `FAILED` we are retrying to payment. Contact Vipps for more information if this failure show up on a `FAILED` charge. |
+
+The user get more information in Vipps regarding why the Charge did not get charged. If the charge has `retryDays` left, recurring wil continue to try and process the charge and notify the user.
```

Comment:
the user **gets** more information..., so if they contact you about failing charges, you should refer them to Vipps.